docs/.doxygen/Doxyfile

```diff
@@ -0,0 +1,59 @@
+# Auto-generated doxygen configuration file
+
+BUILTIN_STL_SUPPORT = YES
+
+CALLER_GRAPH = YES
+
+CALL_GRAPH = YES
+
+ENUM_VALUES_PER_LINE = 1
+
+EXCLUDE_PATTERNS = ../../src/targets/gpu/kernels ../../src/targets/gpu/device
+
+EXTRACT_ALL = YES
+
+FULL_PATH_NAMES = YES
+
+GENERATE_LATEX = NO
+
+GENERATE_TREEVIEW = YES
+
+GENERATE_XML = YES
+
+INCLUDE_PATH = ../../src/include ../../src/targets/cpu/include ../../src/targets/gpu/include
+
+INPUT = ../../src
+
+MACRO_EXPANSION = YES
+
+OUTPUT_DIRECTORY = docBin
+
+PREDEFINED = DOXYGEN
+
+PROJECT_NAME = MIGraphX
+
+RECURSIVE = YES
+
+REFERENCED_BY_RELATION = YES
+
+REFERENCES_LINK_SOURCE = YES
+
+REFERENCES_RELATION = YES
+
+SEARCH_INCLUDES = YES
+
+SORT_MEMBERS_CTORS_1ST = YES
+
+SOURCE_BROWSER = YES
+
+STRIP_FROM_INC_PATH = ../../src/include ../../src/targets/cpu/include ../../src/targets/gpu/include
+
+STRIP_FROM_PATH = 
+
+USE_PDFLATEX = NO
+
+WARN_LOGFILE = DoxygenWarningLog.txt
+
+DOT_PATH = "/usr/bin"
+
+HAVE_DOT = YES
```

Comment:
This should only be set to YES if dot is installed.

Reply:
Fixed in [77fb290](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/1727/commits/77fb2904357c7f8a8bb911d1d905a5258bbe7a2c)